docs/source/contribution.rst

```diff
@@ -154,11 +154,11 @@ Note that we are using pytest and mock package for testing, so install them befo
 
 In order to run unit tests at the repository root, you first have to build Cython files in place by running the following command::
 
-  $ python setup.py develop
+  $ pip install .
```

Comment:
I think this command does not build Cython files *in place*. It should be `pip install -e .`.

Reply:
How about removing "in place" instead?
According to offline discussion with @beam2d, it seems `pip install -e .` is incompatible with `pip install .` and perhaps we should not encourage using `-e`.

Reply:
https://github.com/pypa/setuptools/issues/960

Reply:
I think the experiment result above is for [namespace packages](https://packaging.python.org/guides/packaging-namespace-packages/) (i.e., share the same package namespace from multiple projects.)
I'm always using `pip install -e .` and I think it is compatible. When I run `pip install -e .` then  `pip install .` (and vice versa), `pip` uninstalls the previous one and installs the new one.

Moreover, suggesting contributors to install unstable codebase (in this context s/he is currently writing a code to contribute) to global `site-packages` environment to run unit tests is not good I think.

Reply:
OK, so I think it's ok to write `-e`.

Reply:
Done.